XamlControlsGallery/ControlPages/CommandBarPage.xaml.cs

```diff
@@ -45,12 +69,14 @@ private void AddSecondaryCommands_Click(object sender, RoutedEventArgs e)
 
             if (PrimaryCommandBar.SecondaryCommands.Count == 1)
             {
+                MultipleButtons = true;
```

Comment:
This feedback is really minor, but I would love to see this line at the very end of the If statement. Right now, it reads strangely as the very first line of the condition when exactly one button is visible. Would feel more logical after all the buttons were added to the secondary commands.

Reply:
Oh yes, that makes more sense. I will change that. Thank you for that feedback ! :smile: